docs/develop-load-manager.md

```diff
@@ -207,3 +103,216 @@ then all machines are approximately overloaded. In the case in which a broker's
 threshold, that broker is not considered for bundle assignment. If all brokers are overloaded, the bundle is randomly
 assigned.
 
+## Enablement
+
+You can enable a simple, modular, or extensible load manager by following the steps below.
+
+### Enable simple load manager
+
+You can enable the simple load manager using one of the following methods: 
+
+- Method 1
+
+  Update the value of [loadManagerClassName](https://github.com/apache/pulsar/blob/782e91fe327efe2c9c9107d6c679c2837d43935b/conf/broker.conf#L1309) to `org.apache.pulsar.broker.loadbalance.impl.SimpleLoadManagerImpl` in `conf/broker.conf`.
+
+- Method 2
+  
+  Use the `pulsar-admin` tool. 
+
+   ```shell
+   pulsar-admin brokers update-dynamic-config \
+   --config loadManagerClassName \
+   --value org.apache.pulsar.broker.loadbalance.impl.SimpleLoadManagerImpl
+   ``` 
+
+  :::note 
+  
+  Any mistakes in specifying the load manager will cause Pulsar to default to `SimpleLoadManagerImpl`.
```

Comment:
Out of curiosity, do you mean if users do not configure the load manager type correctly, Pulsar will use `SimpleLoadManagerImpl`?

Reply:
yes